test/e2e/features/regression/extension.feature

```diff
@@ -79,4 +79,24 @@ Feature: Frictionless Browser Extension Modal
 
   Examples:
       | Verb              | File                |
-      | request-signature | test-files/test.pdf |
\ No newline at end of file
+      | request-signature | test-files/test.pdf |
+
+    @MWPW-144462 @regression @extension @headed
+      Scenario Outline: PDF should be displayed inline
+      Given I go to the DC page '<urlPath>'
+      Then I should see inline PDF viewer
+
+      Examples:
+      | urlPath|
+      | acrobat/business/reports/sdk/adobe-acrobat-microsoft-365-brief.html|
+
+    @MWPW-144263 @regression @extension @headed
```

Comment:
Hi @saragajic11, the Acrobat extension needs to be installed prior to running the test. Otherwise, the PDF will only display in the default PDF reader of the browser:
<img width="1792" alt="image" src="https://github.com/adobecom/dc/assets/117126357/5cf98c38-fa63-4b9c-aaf8-78ca860a4cf9">

The test should verify that the PDF displays in the Acrobat PDF reader extension like this:
<img width="1566" alt="image" src="https://github.com/adobecom/dc/assets/117126357/f9efcfba-7171-458a-bb4d-d0d26ecd90f8">